interface/src/index.html

```diff
@@ -15,12 +15,29 @@
     <!-- font awesome from BootstrapCDN -->
     <link href="http://netdna.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.css" rel="stylesheet">
 
+
+    <!-- Required CSS for ui-slider -->
+    <link rel="stylesheet"
+          href="https://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/themes/smoothness/jquery-ui.css">
+
     <!-- compiled CSS --><% styles.forEach( function ( file ) { %>
     <link rel="stylesheet" type="text/css" href="/static/<%= file %>" /><% }); %>
 
+    <!-- Required js for ui-slider
+         FIXME: Why adding these two libraries in build.config.js does not work?
```

Comment:
I started trying to write out the reasons we are trying to avoid a jQuery dependency, but this article does an excellent job of explaining my thoughts on trying to combine jQuery with AngularJS code:  http://tech.zumba.com/2014/08/02/angularjs-forget-jquery/

I would like for us to think of jQuery as a "legacy" dependency. The details are complex because jQuery is indeed an excellent tool and even Angular embeds a stripped-down version they call jQuery-lite: https://docs.angularjs.org/misc/faq#does-angular-use-the-jquery-library- . At a fundamental level, however, what we are trying to do by using AngularJS in our system architecture is to get away from this "old" way of doing things.

Some people (including the developer of `ui-slider`, who lists "AngularUI team" in the LICENSE file) continue to use jQuery as part of their Angular projects. That is fine for them, but not for us. I should also point out that the README for `ui-slider` says "This is WIP so expect breaking changes", which is another good reason to avoid that component entirely apart from its jQuery dependency. The developer is practically advertising that it's not ready for production.

It is possible to implement a slider without requiring jQuery, and there are several examples I found via Google. The one that looked most full featured and actively supported was http://angular-slider.github.io/angularjs-slider/ . Source code here: https://github.com/angular-slider/angularjs-slider .

Please have a look at that and see if it would meet our needs for the slider here.